irohad/main/application.cpp

```diff
@@ -675,10 +675,27 @@ Irohad::RunResult Irohad::initMstProcessor() {
 }
 
 Irohad::RunResult Irohad::initPendingTxsStorage() {
+  using PreparedTransactionType =
```

Comment:
Could we use `PreparedTransactionDescriptor` alias from `pending_txs_storage_impl.hpp`?